### DISCUSSION TITLE
2024 4Q <성장 수집가> 김선빈의 OKR

### DISCUSSION THREAD

**Discussion by SunandBean** [Pseudo-Lab/growth-collector#18] [우리들의 성장 목표!]
O: 내가 공부한 지식으로 누군가에게 도움을 주는 시도를 많이 해보기

- KR1: AI / 자동화에 대한 지식으로 도움 주기
- KR2: 부동산 구매 경험으로 도움 주기
- KR3: 직장과 커리어에 대한 경험으로 도움 주기

**Comment by SunandBean** [comment]
### 기간
2024.11.01 (금) ~ 14 (목)

### 테마
나눔을 위한 채움의 기간!

### Action Item
1. 자동화 도구로 새로운 사례들 테스트 하기 (SNS, 블로그 자동 업로드 등)
2. 부동산 임차에 대한 공부

**Reply by yunhee1** [reply]
파이팅입니다!

**Reply by soncy0311** [reply]
남은 4일도 화이팅입니다!

**Reply by SunandBean** [reply]
[make.com을 이용한 워드프레스 자동 포스팅](https://youtu.be/qH5uMm_hFIE?si=Vs7U1RhRtQ6KmgDV) 따라서 구성하기 완료!

**Reply by SunandBean** [reply]
[대출 껴있는 지금 살고 있는 집을 전세로 내놓을 때 체크해야할 것들](https://www.perplexity.ai/search/daecul-ggyeoissneun-jigeum-sal-JynuLXLfSX2RB2x_tCo9gA#0) 조사 및 공부 완료!

**Comment by SunandBean** [comment]
### 기간
2024.11.14 (목) ~ 27 (수)

### 테마
이관, 임차 시도

### Action Item
1. 기존 코드 기반 자동화를 n8n으로 옮기기 - 유튜브 요약 봇
2. 부동산 임차에 대한 여러 방법 공부 및 부동산 가서 물어보기

**Reply by SunandBean** [reply]
11월 18일 (월) 
- 기존 코드 기반 자동화를 n8n으로 옮기기 - 유튜브 요약 봇 : 완료!
  - AI 모델도 Gemini 에서 gpt-4o-mini 로 변경, 이유는 Gemini는 여러모로 마사지를 해도 시킨대로 동작하지 않는 경우가 많아서!

11월 21일 (목)
- 부동산 가서 전월세 임차 관련해서 물어보기 : 완료!
  - 하... 나가는 집만 팔리는 시기라고 하네...

11월 22일 (금)
- 다른 임차 방법들 알아보기 (삼삼엠투, 에어비앤비) : 완료!
  - 삼삼엠투: 단기임대 계약 플랫폼, 일주일 단위로 계약 진행.
  - 에어비앤이: 룸쉐어 개념의 숙박 제공 플랫폼, 외국인도시민박업 사업자 등록을 해야만 "합법적으로" 가능

**Reply by SunandBean** [reply]
11월 26일 (화)
- 월세 보증금 관련 조사 - 소액 임차인 우선 변제 금액
  - **근저당권 설정일자 기준**으로 해당 지역에 맞는 소액 임차인 조건과 우선 변제 금액을 적용함
    - 서울의 경우 현재 시점 기준 소액임차인 대상은 1억 6,500만원 이하의 보증금, 우선변제 금액은 최대 5,500만원
  - **대항력을 가진 임차인**이 되어야 함 - 전입신고, 확정일자 필수

**Reply by soncy0311** [reply]
개인적으로 LLM 관련해서는 점수 계산, 평가 관련한 테스크를 제외하고는 gpt-4o-mini 프롬프트 엔지니어링으로 다 해결되는 듯 싶습니다!! 참고차 말씀드립니다!

**Reply by SunandBean** [reply]
11월 27일 (수)
- 적정 월세 파악을 위한 조사
  - [전월세전환율](https://kosis.kr/statHtml/statHtml.do?orgId=408&tblId=DT_30404_N0010)에서 지역별 전환율 확인 가능
  - [전월세전환율 계산기](https://xn--989a00af8jnslv3dba.com/%EC%A0%84%EC%9B%94%EC%84%B8%EC%A0%84%ED%99%98)로 얼마에 이 집을 내놓을 수 있는지 확인할 수 있음

**Reply by SunandBean** [reply]
> 개인적으로 LLM 관련해서는 점수 계산, 평가 관련한 테스크를 제외하고는 gpt-4o-mini 프롬프트 엔지니어링으로 다 해결되는 듯 싶습니다!! 참고차 말씀드립니다!

돈내기 싫어서 Gemini 썼던건데... 이젠 가격도 싸졌으니 그냥 gpt-4o-mini로 쓰기로 했습니다...ㅋㅋㅋ

**Comment by SunandBean** [comment]
### 기간
2024.12.02 (월) ~ 15 (일)

### Action Item
- [x] 임차 계약 마무리
- [x] 임차 관련해서 고려해야하는 부분들 공부하기
- [x] Windsurf로 뭐라도 해보기
- [x] 이사 관련 일정 정리 및 예약

**Reply by rlaalsrl0922** [reply]
임차 꿀팁 있으시면 공유 받들겠습니다 💪

**Reply by soncy0311** [reply]
저는 cursor도 안쓰고 windsurf도 안써서 둘의 장단점을 잘은 모르지만, 최근 회사 동료가 cursor쓰다가 windsurf로 넘어갔는데 얼마안되서 다시 cursor로 넘어왔더라고요!

**Reply by SunandBean** [reply]
12월 3일 (화)
- 임차 계약 마무리

**Reply by SunandBean** [reply]
12월 4일 (수) ~ing
- 임대차 계약시 임대인이 계약 신고 필요 -> 행정복지센터 신고 필요
- 생애최초 취득세 감면 조건 미이행에 따른 추징 발생 -> 세무서 신고 필요
- 1가구 1주택의 임대의 경우 임대소득 비과세
- 대출이자 상환 관련 공제 알아보기

**Reply by SunandBean** [reply]
12월 5일 (목)
- 이사 관련 일정 정리 완료
- 이사 관련 업체 견적, 컨택, 예약 일부 완료

**Reply by SunandBean** [reply]
12월 7일 (토)
- 이사 관련 일정 확정
- 이사 업체 방문 견적 및 계약서 작성 완료

**Reply by SunandBean** [reply]
12월 8일 (일)
- Windsurf 로 어플 개발 완료 및 플레이스토어 등록 -> 명의 확인 절차 진행 중
- Windsurf 로 구글 크롬 익스텐션 개발 완료 및 구글 익스텐션 스토어 등록 -> 검토 대기 중

**Reply by SunandBean** [reply]
12월 9일 (월)
- Windsurf 로 어플 개발한 것 디버깅 -> 드디어 실행 제대로 되는 어플 확인 -> 내부 테스트 중

**Reply by SunandBean** [reply]
12월 10월 (화)
- 크롬 익스텐션 출시!

**Reply by SunandBean** [reply]
12월 11일 (수)
- PARA 어플 디버그 -> 세번째 버전코드 가진 소프트웨어 배포 완료 -> 비공개 테스트 신청 완료
- 추징 관련 조사 - 잔금 처리 후 구청 방문해서 처리
